test/test_images/timeout/timeout.go

```diff
@@ -50,6 +49,5 @@ func handler(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	h := network.NewProbeHandler(http.HandlerFunc(handler))
```

Comment:
Why do we not need this anymore?

Reply:
These Serving test images are acting as the user container. They don't need this logic since the queue proxy handles the probe.

Maybe these images were used as part of network conformance in the past where we don't use Knative Services

Reply:
Sgtm